internal/services/gcp_reservation_service.go

```diff
@@ -0,0 +1,121 @@
+package services
+
+import (
+	"net/http"
+
+	"github.com/aws/smithy-go/ptr"
+	"github.com/go-chi/render"
+	"github.com/lzap/dejq"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/jobs"
+	"github.com/RHEnVision/provisioning-backend/internal/jobs/queue"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+)
+
+func CreateGCPReservation(w http.ResponseWriter, r *http.Request) {
+	logger := ctxval.Logger(r.Context())
+
+	var accountId int64 = ctxval.AccountId(r.Context())
+
+	// TODO: meanwhile these values are hardcoded until we will have instance types, sources, ssh endpoints,
+	payload := &payloads.GCPReservationRequestPayload{
+		// TODO: The actual value of the pubkey change to pubkey id
+		PubkeyID: "aabramov:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC8w6DONv1qn3IdgxSpkYOClq7oe7davWFqKVHPbLoS6+dFInru7gdEO5byhTih6+PwRhHv/b1I+Mtt5MDZ8Sv7XFYpX/3P/u5zQiy1PkMSFSz0brRRUfEQxhXLW97FJa7l+bej2HJDt7f9Gvcj+d/fNWC9Z58/GX11kWk4SIXaKotkN+kWn54xGGS7Zvtm86fP59Srt6wlklSsG8mZBF7jVUjyhAgm/V5gDFb2/6jfiwSb2HyJ9/NbhLkWNdwrvpdGZqQlYhnwTfEZdpwizW/Mj3MxP5O31HN45aE0wog0UeWY4gvTl4Ogb6kescizAM6pCff3RBslbFxLdOO7cR17 lzap+rsakey+{{$randomInt}}@redhat.com",
```

Comment:
You can simply use the seeded pubkey ID=1. Feel free to seed your key too and use it as ID=2 :-)